docs/fundamentals/apicompat/preview-apis.md

```diff
@@ -0,0 +1,69 @@
+
+    - Marked with <xref:System.Runtime.Versioning.RequiresPreviewFeaturesAttribute>
+    - Marked with <xref:System.Diagnostics.CodeAnalysis.ExperimentalAttribute>
+
+This article explains how to choose between these options and how each of them works.
+
+## .NET runtime previews
+
+With the exception of release candidates (RCs) with a go-live license, preview versions of the .NET runtime and SDK aren't supported.
+
+As such, any APIs added as part of a .NET preview are considered subject to change, based on feedback the previews receive. To use a .NET runtime preview, you need to explicitly target a newer framework version. By doing so, you implicitly express consent to consume APIs that might change.
+
+## Prerelease NuGet packages
+
+NuGet packages can either be stable or marked as prerelease, that is, the package has a prerelease suffix. For example, `System.Text.Json 9.0.0-preview.2.24128.5` has a prerelease suffix of `preview.2.24128.5`.
+
+Package authors generally don't support prerelease packages and use them as a means to collect feedback from early adopters.
+
+When installing packages, either via the CLI or UI, you generally have to explicitly indicate whether you want to install prerelease, again implicitly expressing consent to consume APIs that might change.
+
+## `RequiresPreviewFeaturesAttribute`
+
+Since .NET 6, the platform includes the <xref:System.Runtime.Versioning.RequiresPreviewFeaturesAttribute> attribute.
+
+This attribute is used for APIs that require preview behaviors across the stack, including the runtime, the C# compiler, and libraries. When you consume APIs marked with this attribute, you'll receive a build error unless your project sets `<EnablePreviewFeatures>true</EnablePreviewFeatures>`. Setting that property to `true` also sets `<LangVersion>Preview</LangVersion>`, allowing the use of preview language features.
+
+The .NET team used the <xref:System.Runtime.Versioning.RequiresPreviewFeaturesAttribute> attribute in .NET 6 to test out generic math, because it required static interface members, which were in preview at the time.
+
+## `ExperimentalAttribute`
+
+.NET 8 added <xref:System.Diagnostics.CodeAnalysis.ExperimentalAttribute>, which doesn't require any runtime or language preview features and simply indicates that a given API isn't stable yet.
+
+When building against an experimental API, the compiler will produce an error. Each feature that is marked as experimental has its own separate diagnostic ID. To express consent to using them, you suppress the specific diagnostic. You can do that via any of the means for suppressing diagnostics, but the recommended way is to add the diagnostic to the project's `<NoWarn>` property.
+
+Since each experimental feature has a separate ID, consenting to using one experimental feature doesn't consent to using another.
+
+## Choose between the options
+
+Library developers should only use prerelease NuGet packages or mark APIs with `[Experimental]`:
```

Comment:
You're using "you" in some parts and specific groups like "library developers" "package authors" elsewhere. Either change to 3rd person throughout, or make sure it's clear which group "you" refers to